web/components/FcDetailsStudyRequest.vue

```diff
@@ -6,9 +6,9 @@
         <v-col cols="8">
           <v-text-field
             v-model="internalValue.serviceRequestId"
+            label="Service Number"
             :messages="[OPTIONAL.text]"
-            outlined
-            placeholder="Service Number">
+            outlined>
```

Comment:
realized that we still have a placeholder instead of a label in `FcCommentsStudyRequest`, please fix.